packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -66,7 +67,7 @@ class FlutterExtension {
      * Specifies the relative directory to the Flutter project directory.
      * In an app project, this is ../.. since the app's Gradle build file is under android/app.
      */
-    String source = "../.."
+    String source
```

Comment:
just a note:

If we want to have a default value, we should also have a test that checks that the app builds fine if `flutter { source "../.." }` is missing from `android/app/build.gradle`

fyi @Gustl22

Reply:
Thank you @bartekpacia. I actually don't propose to remove the `source` property from the `build.gradle`. I propose to test and/or remove it in a separate step. Should I open an issue?